src/tools/manifest2proto.ts

```diff
@@ -57,23 +58,69 @@ function encodePayload(payload: {}): Uint8Array {
 }
 
 async function particleSpecToProtoPayload(spec: ParticleSpec) {
+  let claims = [];
+  const connections = await Promise.all(spec.connections.map(async cs => {
+    const directionOrdinal = DirectionEnum.values[cs.direction.replace(/ /g, '_').toUpperCase()];
+    if (directionOrdinal === undefined) {
+      throw Error(`Handle connection direction ${cs.direction} is not supported`);
+    }
+    const proto = {
+      name: cs.name,
+      direction: directionOrdinal,
+      type: await typeToProtoPayload(cs.type)
+    };
+    const claimsProto = claimsToProtoPayload(cs, proto);
```

Comment:
If you wanted, you could inline `proto` inside the call to `claimsToProtoPayload`

Reply:
I am actually returning `proto` at the end?